exodus_lambda/functions/origin_request.py

```diff
@@ -374,6 +364,60 @@ def handler(self, event, context):
                         return response
 
                     return out
+
+    def handler(self, event, context):
+        # pylint: disable=unused-argument
+        request = event["Records"][0]["cf"]["request"]
+
+        if not self.validate_request(request):
+            return {"status": "400", "statusDescription": "Bad Request"}
+
+        self.logger.debug(
+            "Incoming request value for origin_request",
+            extra={"request": request},
+        )
+
+        request["uri"] = unquote(request["uri"])
+        original_uri = request["uri"]
+
+        if request["uri"].startswith("/_/cookie/"):
+            return self.handle_cookie_request(event)
+
+        uri = self.resolve_aliases(request["uri"])
+        fallback_uri = self.resolve_aliases(
+            request["uri"], ignore_exclusions=True
+        )
+        fallback_uri = fallback_uri if uri != fallback_uri else None
```

Comment:
It may be helpful to include a comment explaining the purpose/context of "fallback_uri" for posterity.